sierra_adapter/sierra_reader/src/main/scala/uk/ac/wellcome/platform/sierra_reader/services/SierraReaderWorkerService.scala

```diff
@@ -43,9 +55,9 @@ class SierraReaderWorkerService @Inject()(
     process = processMessage
   )
 
-  def processMessage(norificationMessage: NotificationMessage): Future[Unit] =
+  def processMessage(notificationMessage: NotificationMessage): Future[Unit] =
     for {
-      messageString <- Future(norificationMessage.Message)
+      messageString <- Future(notificationMessage.Message)
```

Comment:
We should use `Future.successful` if there isn't an actually blocking operation (i know this is how it was before but it would be good to fix it anyway)